project/LinkerdBuild.scala

```diff
@@ -126,6 +127,13 @@ object LinkerdBuild extends Base {
 
       val all = projectDir("linkerd/protocol")
         .aggregate(http, mux, thrift)
+
+      val benchmark = projectDir("linkerd/protocol/benchmark")
+        .dependsOn(core, Router.http)
```

Comment:
I don't think you import anything from router/http inside the linkerd/protocol/benchmark project -- maybe just remove line 132?